poms/bom/pom.xml

```diff
@@ -2508,6 +2508,11 @@
                 <artifactId>camel-telegram</artifactId>
                 <version>${camel.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel</groupId>
+                <artifactId>camel-test-junit5</artifactId>
```

Comment:
Not sure if we want to have this in the BOM yet given that https://github.com/apache/camel-quarkus/issues/3511 is unresolved. WDYT @ppalaga?

The convention used in these `QuarkusUnitTest` type tests is to create a `RouteBuilder` impl in the test class and it will get picked up automatically by the CQ bootstrap.

Reply:
OK, I remove it and will take a look at #3511 .